src/controllers/accommodation.js

```diff
@@ -113,5 +113,21 @@ export default class accomodationFacility {
       return res.status(500).json({ status: 500, errorMessage: error });
     }
   }
+
+  static async mostTraveled(req, res) {
+    const traveledCentre = await Bookings.findAll({
+      include: [
+        {
+          model: models.Accommodation,
```

Comment:
Please use Accommodation from import.

Reply:
How I forgot this I thought I did it let do it and rebase also